src/jdk.crypto.cryptoki/share/classes/sun/security/pkcs11/SunPKCS11.java

```diff
@@ -470,6 +479,17 @@ private static void d(String type, String algorithm, String className,
         register(new Descriptor(type, algorithm, className, aliases, m));
     }
 
+    private static void d(String type, String algorithm, String className,
+            int[] m, int[] requiredMechs) {
+        register(new Descriptor(type, algorithm, className, null, m,
+                requiredMechs));
+    }
+    private static void dA(String type, String algorithm, String className,
+                           int[] m, int[] requiredMechs) {
+        register(new Descriptor(type, algorithm, className,
+                getAliases(algorithm), m, requiredMechs));
+    }
+
```

Comment:
nit: move this to after the other dA(String, String, String, int[]) below

Reply:
Good